epregressions/runtests.py

```diff
@@ -372,7 +375,8 @@ def run_build(self, build_tree):
         # `apply_async` approach I am using.  Blech.  Once again, on Windows, this means it will partially not be
         # multithreaded.
         if self.number_of_threads == 1 or frozen and system() in ['Windows', 'Darwin']:  # pragma: no cover
-            self.my_print("Ignoring num_threads on frozen Windows/Mac instance, just running with one thread.")
+            if self.number_of_threads > 1:
+                self.my_print("Ignoring num_threads on frozen Windows/Mac instance, just running with one thread.")
```

Comment:
Only print this warning message if you are actually trying to run multithreaded.  Previously it would show up even if you intentionally ran a single thread.